app/templates/event/partials/edit_event_details_tab.html

```diff
@@ -33,7 +33,19 @@
             <option value="blue">Blue</option>
         </select>
     </div>
-
+    <div class="user_categories">
+        <select name="the_categories">
+            {% if categories_list[0] is defined %}
+                <option style="color:{{categories_list[0]['color']}}" value="{{categories_list[0]['name']}}" selected>{{categories_list[0]['name']}}</option>
```

Comment:
Pay attention that wrong format can cause here lots of problems..

The format you chose is "#aabbcc", right?
I can add validation in BE, please consider FE...

Reply:
I started to add a validation in BE side, and I think we have some issue -
`#` has a special meaning in URLs (it breaks tests)

So, I suggest that FE will send the color in **"aabbcc"** format, and that's what we will store in the db.
It means that here you need to add `#` before, something like:
```
style="color:#{{categories_list[0]['color']}}"
```
Is that ok with you?

Reply:
Validation in BE: https://github.com/PythonFreeCourse/calendar/pull/251